multi-int-store/src/test/java/com/connexta/multiintstore/SolrTests.java

```diff
@@ -93,16 +95,21 @@ public SolrClientConfiguration testSolrClientConfiguration() {
   private String endpointUrlRetrieve;
 
   @Before
-  public void before() {
+  public void before() throws IOException, SolrServerException {
     when(mockAmazonS3.putObject(any(PutObjectRequest.class)))
         .thenReturn(mock(PutObjectResult.class));
+
+    solrClient.deleteByQuery(SOLR_COLLECTION, "*");
+    solrClient.commit(SOLR_COLLECTION);
```

Comment:
We should not need to clear solr between itests. I believe that it is better to write tests that do not have the assumption that Solr is empty, because in a production instance, Solr will never be empty. In each test, we can get the id from the original store request or we can use a fake id to test an id that doesn't exist.